frontend/components/LeaveModal/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Text, Image, View, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+
+import FontAwesome5 from 'react-native-vector-icons/FontAwesome5';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import Svg, {
+    Circle,
+    Ellipse,
+  } from 'react-native-svg';
+
+export type Props = {};
```

Comment:
This Props type looks sketchy. Lol! maybe you can specify what's inside of it.
Why export tho, u're using it somewhere else too?

Reply:
Thanks, will [specify types](https://alanyan.atlassian.net/browse/SG-56) soon! Probably don't need to export them but I remember wanting to make the modals reusable at some point...